src/dlgAboutDialog.cpp

```diff
@@ -1103,17 +1106,157 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
 
 QString dlgAboutDialog::createBuildInfo() const
 {
+#if defined(Q_OS_WIN32)
+    // The build environment is for Windows one - which could be run
+    // native on a 32-bit or 64-bit CPU or inside the WOW64 sub-system on a
+    // 64-bit one:
+
+    auto hProcess = GetCurrentProcess();
+    BOOL value = false;
+    std::optional<bool> isWow64Process;
+    auto result = IsWow64Process(hProcess, &value); // hProcess is a "HANDLE"
+    if (!result) {
+        // Failed to work - so there is no value to assign to isWow64Process:
+        qWarning().nospace().noquote() << "dlgAboutDialog::createBuildInfo() WARNING - IsWow64Process(...) failed, WOW64 status unknown.";
+    } else {
+        isWow64Process = static_cast<bool>(value);
+    }
+    if (Q_UNLIKELY(QLatin1String(qVersion()) != QLatin1String(QT_VERSION_STR))) {
+        // Originally QLatin1String::compare(...) was to be used but, despite
+        // the Qt documentation claiming that that method was introduced in
+        // Qt 5.14 it lies; CI builds using Qt version 5.14.2 were borking
+        // and giving an: "error: ‘class QLatin1String’ has no member named
+        // ‘compare’" message instead!
+        return qsl("<table border=\"0\" style=\"margin-bottom:18px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n"
+                   "<tr><td colspan=\"2\" style=\"font-weight: 800\">%1</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%2<td>%3</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%4</td><td>%5</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%6</td><td>%7</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%8</td><td>%9</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%10</td><td>%11</td></tr>\n"
+                   "</table>")
+                .arg(tr("Technical information:"), // %1
+                     tr("Version"), // %2
+                     mudlet::self()->scmVersion, // %3
+                     tr("OS"), // %4
+                     QSysInfo::prettyProductName(), // %5
+                     /*: This is shown for 32-Bit Windows builds when run on a
+                      *64-Bit OS. \"WoW64\" stands for WindowOnWindows64.
+                      */
+                     isWow64Process.has_value() ? (isWow64Process.value() ? tr("CPU (WoW64)")
+                     /*: This is shown for 32-Bit or 64-Bit Windows builds when
+                      *a Windows OS of the same size. It is the opposite case
+                      *to that when \"WoW64\" is included - in those cases a
+                      *32-Bit application is run on 64-Bit hardware via an
+                      *extra WindowOnWindows64 software layer.
+                      */
+                                                                          : tr("CPU (Native)"))
```

Comment:
I think it would be nice to know if you are running a 32bit or a 64bit version of Mudlet even if it is the right version for you.

Reply:
Well that is what "Native" is intended to, *succinctly*, say. However - due to the limitations on the original `IsWow64Process(...)` it only reports whether WoW64 is in use or not - for more information - including for Windows on Arm devices - you need `IsWow64Process2(...)` but that is not available in the MINGW64 environment.

Reply:
To put this into context, if you downloaded Mudlet and someone asks you are you using a 32bit or a 64bit one, the dialog just saying "native" will not help you or the person who is asking you :sweat_smile:

Reply:
Well, I suppose you'd need to know which version of Windows they are using - let me think about that.

Reply:
I reckon the simplest thing to do would be to just show if the running Mudlet is 32bit or a 64bit application.

Reply:
I've updated this PR to show the "bitness" if it ***isn't*** running in the WoW64 case.